src/reference-designs/tkg-edge.md

```diff
@@ -0,0 +1,885 @@
+
+You can use Velero to back up and restore a workload cluster’s current workloads and persistent volumes state, for entire clusters or specific namespaces. You can configure pre- and post-hooks for backup and restore to run custom processes before or after backup and restore events. If some aspects of workload or cluster state are not well-suited to backup/restore, you can exclude them.
+
+For the Tanzu Edge architecture, it is recommended to backup the stateful workloads on the main datacenter. The main datacenter should have S3 storage since Velero requires S3 storage to be used as a target for the backup.
+
+While Velero can be setup manually through its CLI, for fleet management it is recommended to use Tanzu Mission Control and its Data Protection capability to perform such a task.
+
+#### Loss of physical host
+
+The impact of the loss of a host is highly dependent of the workloads that are running on it at the time of the outage. These workloads are going to be automatically restarted by vSphere HA, however minor disruption might occur.
+Here are some example depending on the workload impacted:
+
+- Active NSX ALB Service Engine:
+Applications using loadbalancer might not be available for few seconds until the Standby LB Service Engine takes over.
+- Control Plane node:
+In case of a Kubernetes cluster with a single control plane node, the management of the cluster will be unavailable for few minutes until the control plane is restarted and all the Kubernetes components are back up and running.
+- Worker node with application pods and persistent volumes:
+If the application is deployed in a highly available fashion, there should not be any disruption.
+
+### Centralized control plane
+
+For companies modernizing their application estate, it is not uncommon to have cloud native deployments across many locations. They will run Kubernetes clusters in their core data centers (in the 10s), mini data centers away from the core data center (in the 100s), and thousands of smaller 2 to 3 server deployments at locations like stores, like what we are testing here. To be able to manage at scale this vast Kubernetes estate you need a centralized control plane that can help you deploy, manage, secure and provide insights into kubernetes deployments. For this reason, a centralized control plane in public cloud, private data center, or a hybrid environment is a requirement. One way to solve this is via Tanzu Mission Control.
+
+TMC is a centralized management platform for consistently operating and securing your Kubernetes infrastructure and modern applications across multiple teams and clouds. It can manage and ensure operations across a large fleet of edge Kubernetes clusters consistently with the capabilities described below
```

Comment:
Tanzu Mission Control is a centralized ... consistently with the following capabilities: